java-util/src/main/java/io/druid/java/util/common/parsers/ParserUtils.java

```diff
@@ -94,4 +94,25 @@ public static String stripQuotes(String input)
     }
     return input;
   }
+
+  public static String replace(String text, String searchString, String  replacement)
```

Comment:
Also, if this method is needed, it would be better to add some tests for this method.